com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Mask/ColorMaskNode.cs

```diff
@@ -11,10 +11,7 @@ internal class ColorMaskNode : IStandardNode
         public static FunctionDescriptor FunctionDescriptor => new(
             Version,
             Name,
-@"
-{
-    Out = saturate(1 - (distance(MaskColor, In) - Range) / max(Fuzziness, 1e-5));
-}",
+@"  Out = saturate(1 - (distance(MaskColor, In) - Range) / max(Fuzziness, 1e-5));",
```

Comment:
Very small nit pick - body code that's just one line doesn't need the @ symbol in front of it and can be tabbed over so that it lines up with Version, Name, and the ParameterDescriptors..